mkt/extensions/views.py

```diff
@@ -58,7 +57,9 @@ def create(self, request, *args, **kwargs):
         if not file_obj:
             raise exceptions.ParseError(_('Missing file in request.'))
 
-        self.validate_upload(file_obj)  # Will raise exceptions if appropriate.
+        # Will raise exceptions if appropriate.
+        ExtensionValidator(file_obj).validate()
```

Comment:
Maybe add a quick view test to make sure we are indeed calling `ExtensionValidator().validate()` (you can just mock it and check that is has been called with the file-like object)